Java/Assignment_4/KYC.java

```diff
@@ -0,0 +1,39 @@
+package ASsignment_4;
+
```

Comment:
Having a problem statement file or link would be good.

Reply:
Done !